packages/lesswrong/components/messaging/NewConversationButton.tsx

```diff
@@ -54,16 +56,11 @@ const NewConversationButton = ({ user, currentUser, children, templateCommentId
     conversationExists ? undefined : void newConversation(search);
   }
   
-
-  if (currentUser) {
-    return (
-      <div onClick={existingConversationCheck}>
-        {children}
-      </div>
-    )
-  } else {
-    return <Components.Loading />
-  }
+  return (
+    <div onClick={currentUser ? existingConversationCheck : () => openDialog({componentName: "LoginPopup"})}>
+      {children}
+    </div>
+  )
```

Comment:
I also made it so that the "message" and "subscribe" buttons on the user profile are visible even to logged out users (clicking them prompts the user to sign up / login). I'll probably be doing more of this going forward, since many (most?) of our users don't have an account yet, and we want to expose these features to them.